components/page/Layout.tsx

```diff
@@ -0,0 +1,44 @@
+import Meta from '@components/page/Meta'
+
+import Link from 'next/link'
+import { Text, TextContent, Brand, Page, PageHeader, PageHeaderTools, PageSection, PageSectionVariants } from '@patternfly/react-core'
+
+import mainPageLogo from '../../public/images/mainPageLogo.svg'
+
+interface Props {
+  children: React.ReactNode
+}
+
+const Layout = ({ children }: Props): React.ReactElement => {
+  const Header = (
+    <PageHeader
+      logo={
+        <Brand src={mainPageLogo} alt="Devfile Registry Logo" className="h-full"/>
```

Comment:
interestingly the logo is a link but it doesnt redirect anywhere 🤔

Reply:
fixed